src/VisualNode.tsx

```diff
@@ -1,98 +1,73 @@
-import { useRef, useState } from 'react'
-import { useDrag, useDrop, DropTargetMonitor } from 'react-dnd'
-import { useSelectNodeIndex, useWorkflow } from './store'
 import { GripVertical, X } from 'react-bootstrap-icons'
-import { DragItem } from './types'
-import { nodeWidth } from './constants'
+import { CSS } from '@dnd-kit/utilities'
+import { useSortable } from '@dnd-kit/sortable'
+import { useDraggingWorkflowNodeState, useSelectNodeIndex, useWorkflow } from './store'
+import classes from './VisualNode.module.css'
 
 interface IProp {
   id: string
   index: number
+  code: string
 }
```

Comment:
id/code was adjusted to type/ID (in 4b445b1 and 1d8dccc) — should this be included? 
(+ other instances in this file)

Reply:
Good catch, that component does not use IWorkflowNode type so I missed it.